src/main/java/core/basesyntax/dao/impl/FruitDaoImpl.java

```diff
@@ -0,0 +1,66 @@
+package core.basesyntax.dao.impl;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.model.Fruit;
+import java.util.Map;
+
+public class FruitDaoImpl implements FruitDao {
+    private static final String NULL_INPUT_MESSAGE = "Invalid operation. "
+            + "Input data should not be null!";
+    private static final String ADD_FIRST_FAILURE_MESSAGE = "Product {%s} "
+            + "already exist in storage!";
+    private static final String ADDING_FAILURE_MESSAGE = "Product {%s} "
+            + "does not exist in storage!";

```

Comment:
Where do you use these constants?

Reply:
@YevheniiVlasenko I deleted methods with them while fixed bags :)